### PR TITLE
Add section on block storage encryption

### PR DESCRIPTION
**Pull request by andrea-manzi**
# Summary

translated from https://grand-est.fr/support-utilisateurs/documentation-en-ligne/guide-dutilisation-dopenstack/#disk-encryption 
---

**Related issue :**

### CODE REVIEW

**Comment by gwarf** [comment]
I'm a bit concerned by reading "translated from" :)
Have we their authorisation for reusing their content? Have you any info about licence used for their content and so on?

**Comment by andrea-manzi** [comment]
> I'm a bit concerned by reading "translated from" :) Have we their authorisation for reusing their content? Have you any info about licence used for their content and so on?

i got the link from Jerome as that is the SCIGNE documentation, should i check with him?

**Comment by gwarf** [comment]
> i got the link from Jerome as that is the SCIGNE documentation, should i check with him?

Yes please, you could even propose him to comment here that he is fine :)

**Comment by gwarf** [comment]
Just to log that we got the confirmation from @Pansanel that it's fine to reuse/adapt this content.

File: content/en/users/online-storage/block-storage/_index.md [hunk 1]
```diff
@@ -594,38 +585,169 @@ vdb    252:16   0  10G  0 disk
...
+```
+
+Step 5: Check available space (this may be slightly different from what was
+entered during the `openstack volume create` command):
+
+```shell
+$ df -h /storage1
+Filesystem            Size  Used Avail Use% Mounted on
+/dev/mapper/storage1  2.0G  6.0M  1.9G   1% /storage1
+
```
Comment (gwarf) [inline content/en/users/online-storage/block-storage/_index.md:721]:
```suggestion
```

**Comment by gwarf** [comment]
The user documentation got reorganised via #414, can you please resolve the conflicts in your PR?

**Comment by andrea-manzi** [comment]
@gwarf rebase done

**Review by gwarf — APPROVED** [review]
LGTM.
This can be further refined/extender later, as needed.

**Pull request merged**